plugins/module_utils/mh/base.py

```diff
@@ -59,4 +59,7 @@ def run(self):
         self.__init_module__()
         self.__run__()
         self.__quit_module__()
-        self.module.exit_json(changed=self.has_changed(), **self.output)
+        output = self.output
+        if 'failed' not in output:
+            output['failed'] = False
```

Comment:
Only the `CmdMixin` adds `rc` to the output, so it might make more sense to do this in that mixin only.

Reply:
I don't see where in the `CmdMixin` this can be added.

Reply:
Yeah, come to think of it, this will not fit easily in there. This solution still bothers me a bit - like it is not in the right place - but I don't have a better solution off the bat right now, and I reckon we should fix it sooner rather than later.